packages/itwinui-css/src/tile/tile.scss

```diff
@@ -350,8 +356,4 @@
   @at-root &.iui-folder .iui-tile-thumbnail {
     box-shadow: var(--iui-size-2xs) 0 0 var(--iui-color-foreground-#{$status});
   }
-
-  .iui-tile-name-label {
-    color: var(--iui-color-foreground-#{$status});
-  }
```

Comment:
Are these lines removed because it gets the color from the `.iui-tile-content .iui-tile-name`?

![image](https://user-images.githubusercontent.com/45748283/192820510-0093d4db-fd24-4c6f-9e7d-3fe768a7c42c.png)

Reply:
Correct, the tile name is getting it's color from `var(--_iui-tile-title-text-color)`.

Previously, we set `.iui-tile-name`'s color and then would override `.iui-tile-name-label` when the status modifier was applied.
```scss
.iui-tile-name {
  color: var(--iui-color-foreground-2);
}
.iui-tile.iui-positive .iui-tile-name-label {
  color: var(--iui-color-foreground-positive);
}
```

Now , we set `.iui-tile-name` to a component specific CSS variable mapped to generic CSS variable, allowing us to then change the component specific CSS variable instead.
```scss
.iui-tile-name {
  --_iui-tile-title-text-color: var(--iui-color-foreground-2);
  color: var(--_iui-tile-title-text-color);
}
.iui-tile.iui-positive {
  --_iui-tile-title-text-color: var(--iui-color-foreground-positive);
}
```